docs/testplan/capacity_test_2TierNetwork.md

```diff
@@ -0,0 +1,31 @@
+# Test Objective
+This test aims to track switch power usage, temperature, queue watermark and health state.
+
+# Test Setup
+![Test Setup](./2TierNetwork.png)
+
+1.	The testbed consists of four IXIA traffic generators (synchronized using a time-sync metronome) and five SONiC switches, where the BT1 switch is the Device Under Test (DUT).
```

Comment:
replace all ixia into traffic generator. the snappi-api are platform independent.